org.envirocar.app/build.gradle

```diff
@@ -189,4 +190,6 @@ configurations.configureEach {
             }
         }
     }
+    // TODO(alexmercerind): Remove after migration to org.envirocar.map.
+    exclude module: "mapbox-android-core"
```

Comment:
Added due to duplicate classes conflict between Mapbox v9 (from `org.envirocar.app`) & Mapbox v11 (from `org.envirocar.map`).